src/test/java/org/example/P20_SumOfDigitsTest.java

```diff
@@ -0,0 +1,15 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class P20_SumOfDigitsTest {
+
+    @Test
+    void digitsSum() {
+        int number = 222;
```

Comment:
`222` is not such a good test case because all digits are the same. try something with different digits like `135`

Reply:
done